src/main/java/mate/academy/internetshop/service/BucketService.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.internetshop.service;
+
+import java.util.List;
+import java.util.Optional;
+
+import mate.academy.internetshop.model.Bucket;
+import mate.academy.internetshop.model.Item;
+
+public interface BucketService {
+
+    void addItem(Bucket bucket, Item item);
+
+    Bucket addItem(Long bucketId, Long itemId);
+
+    void create(Bucket bucket);
+
+    List<Item> getAllItems(Bucket bucket);
+
+    Optional<Bucket> get(Long bucketId);
+
+    Bucket update(Bucket bucket);
+
+    boolean delete(Long bucketId);
```

Comment:
```suggestion
    boolean deleteById(Long bucketId);
```

Reply:
I could make a method with a different name, but it is overload, such logic

Reply:
@MateAcademy okay, as u wish